LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-rows-005.html

```diff
@@ -55,11 +55,11 @@
 </grid>
 
 <grid title="Fifth item 3ch row 1">
+  <item style="height:3ch; grid-row:1">5 5</item>
```

Comment:
Could you explain (probably in the commit message) why we need to update the test itself and not just the reference? It's not super clear to me why this test is not valid anymore

Reply:
Updated the commit message. This has to do with the spec says definite items are placed in order. Our implementation relies on the old spec definition which said that the definite items are placed first.

To get around this conundrum these two definitions give the same result if the definite grid items is simply moved to the first spot.